test/examples/secure_agentless_scanning/single/main.tf.OFF

```diff
@@ -0,0 +1,18 @@
+provider "aws" {
+  region = "us-east-1"
+  access_key  = "test"
+  secret_key  = "test"
+
+  endpoints {
+    iam     = "http://127.0.0.1:5000/"
+    sts     = "http://127.0.0.1:5000/"
+    kms     = "http://127.0.0.1:5000/"
+  }
+}
+
+module "single-agentless-scanning" {
+  source            = "../../../..//modules/services/agentless-scanning"
+  trusted_identity  = "arn:aws:iam::123456789012:role/secure-assume-role"
+  external_id       = "external_id"
+  name              = "sysdig"
```

Comment:
don't you need kms_alias and kms_key here as well?

Reply:
perhaps, but I turned the test off because it can't work.